_github-app/LICENSE

```diff
@@ -0,0 +1,15 @@
+ISC License
```

Comment:
we should probably update this?

Reply:
Should actually be deleted no? Just inherit it from the main folder